mandatory/2-function-creation.js

```diff
@@ -4,7 +4,19 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+function isAcceptableUser(userAge, isLoggedIn) {
+  if (userAge >= 18 && isLoggedIn) {
+    return true;
+  }
+  else {
+    return false;
+  }
+}
+
+let userAge = 18;
+let isLoggedIn = true;
```

Comment:
Well done, this look great! I think we dont need to create the variables userAge and isLoggedIn  outside the function and assing them values, the test will provide the values to check the code works.